service_quotas/service_quotas.go

```diff
@@ -74,6 +82,7 @@ type ServiceQuotas struct {
 	region        string
 	quotasService servicequotasiface.ServiceQuotasAPI
 	usageChecks   map[string]UsageCheck
```

Comment:
Maybe it makes sense to rename that to something like `serviceQuotasUsageChecks` now that we have `otherChecks` as well?